internal/host/host.go

```diff
@@ -293,6 +295,44 @@ func (m *Manager) UpdateInventory(ctx context.Context, h *models.Host, inventory
 	return m.updateInventory(ctx, cluster, h, inventoryStr, m.db)
 }
 
+func (m *Manager) ntpSyncedChanged(c *common.Cluster, host *models.Host, inventoryStr string) (bool, error) {
+	prev, err := common.IsNtpSynced(c)
```

Comment:
can't we just validate value from validations list?

Reply:
Maybe we can do it for the old value.  How can we do it for the new value?

Reply:
The more correct way is to do it on the raw data, and not on calculated which might not be in sync with the raw data

Reply:
Agree with new one but old one should be taking from validation list cause it's value is not relevant in the moment we use it. I mean that if previously timestamp was ok, when you will run this validation next time it can return failure even if it was previous value was good in previous run.

Reply:
The old value is relevant since we need to know if to trigger monitoring on this cluster.  The validations is not exact, and depends on when monitoring is invoked.  It is incorrect to use it.